app/qml/settings/MMSettingsController.qml

```diff
@@ -84,13 +91,11 @@ Item {
 
     MMChangelogPage {
       id: changelogPanel
-      onClose: stackview.pop(null)
-      Component.onCompleted: forceActiveFocus()
+
+      onBackClicked: root.back()
+
       model: MM.ChangelogModel {
-        onErrorMsgChanged: function(msg) {
-          changelogPanel.errorDialog.text = msg
-          changelogPanel.errorDialog.open()
-        }
+        onLoadingFailure: __notificationModel.addError( qsTr("Changelog could not be loaded") )
```

Comment:
ErrorDialog is just done by Vitor in https://github.com/MerginMaps/mobile/pull/3110, but it is not merged... how you want to resolve it?